.github/pull_request_template.md

```diff
@@ -1,52 +1,18 @@
 <!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺
 v                               ✰  Thanks for creating a PR! ✰    
-v    Before smashing the submit button please review the checkboxes.
-v    If a checkbox is n/a - please still include it but + a little note why
-v    If your PR doesn't close an issue, that's OK!  Just remove the Closes: #XXX line!
 ☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->
-
+## Which issue does this close?
 Closes: #XXX
+## Purpose of the change
+*Add a description of the overall background and high level changes that this PR introduces*
 
-## What is the purpose of the change
-
-*Add a description of the overall background and high level changes that this PR introduces *
-
-*(E.g.: This pull request improves documation of area A by adding ....*
-
-
-## Brief change log
-
+## Brief change log (include any info relevant to tests))
 *(for example:)*
- 
   - *The metadata is stored in the blob store on job creation time as a persistent artifact*
-  - *Deployments RPC transmits only the blob storage reference*
-  - *Daemons retrieve the RPC data from the blob cache*
-
-
 ## Testing and Verifying
-
-*(Please pick one of the following options)*
-
-This change is a trivial rework / code cleanup without any test coverage.
-
-*(or)*
-
-This change is already covered by existing tests, such as *(please describe tests)*.
-
-*(or)*
-
-This change added tests and can be verified as follows:
-
-*(example:)*
```

Comment:
Do others dislike this? I thought this was great, I just choose the one I want and delete everything else